lib/galaxy/dependencies/dev-requirements.txt

```diff
@@ -171,7 +171,7 @@ trio==0.26.0 ; python_version >= "3.8" and python_version < "3.13"
 tuspy==1.0.3 ; python_version >= "3.8" and python_version < "3.13"
 twill==3.2.5 ; python_version >= "3.8" and python_version < "3.13"
 twine==5.1.1 ; python_version >= "3.8" and python_version < "3.13"
-typing-extensions==4.12.2 ; python_version >= "3.8" and python_version < "3.13"
+typing-extensions==4.9.0 ; python_version >= "3.8" and python_version < "3.13"
```

Comment:
https://github.com/SeleniumHQ/selenium/issues/14278 , should be fixed in selenium==4.23.1